app/views/documents/show.html.erb

```diff
@@ -27,6 +27,14 @@
           <% end %>
         <% end %>
       <% end %>
+      <% if @document.class.has_organisations? %>
```

Comment:
Post review thoughts: could we also save the organisations in a variable and pass them to the name helper? As currently we fetch all the organisations from Publishing API for each organisation to retrieve the name.

Reply:
Yeah, we could memoize the orgs in a helper. I'd forgotten that we make a full call for each run of Organisation.all.